tests/pytest/conftest.py

```diff
@@ -3,14 +3,17 @@
 from django.middleware.locale import LocaleMiddleware
 
 import pytest
-from pytest_socket import disable_socket
+
+# from pytest_socket import disable_socket
 
 from benefits.core import session
 from benefits.core.models import AuthProvider, EligibilityType, EligibilityVerifier, PaymentProcessor, PemData, TransitAgency
+from django.contrib.auth.models import User
 
 
 def pytest_runtest_setup():
-    disable_socket()
+    # disable_socket()
+    pass
```

Comment:
temporary for local unit test running

Reply:
Probably related to this? https://github.com/microsoft/vscode-python/issues/22383

I added a workaround mentioned in that thread to my personal VSCode `settings.json`: https://github.com/thekaveman/dotfiles/commit/453a265767b5fd2481cd6dedad118c2f3cc608c9